docker-compose.yml

```diff
@@ -301,6 +301,9 @@ services:
   snuba-profiling-functions-consumer:
     <<: *snuba_defaults
     command: rust-consumer --storage functions_raw --consumer-group snuba-consumers --auto-offset-reset=latest --max-batch-time-ms 1000 --no-strict-offset-reset --no-skip-write
+  snuba-spans-consumer:
+    <<: *snuba_defaults
+    command: rust-consumer --storage spans --consumer-group snuba-consumers --auto-offset-reset=latest --max-batch-time-ms 1000 --no-strict-offset-reset --no-skip-write
```

Comment:
The reason why we use `snuba-consumers` as the consumer-group is because it used to be an optional param where that was the default, and we needed to keep the default when cutting over to rust consumers. Since this is a new consumer, I think we can use `spans_group` for this instead